cleanrl/sac_continuous_action.py

```diff
@@ -48,22 +51,42 @@
                         help="the wandb's project name")
     parser.add_argument('--wandb-entity', type=str, default=None,
                         help="the entity (team) of wandb's project")
-    
+    parser.add_argument('--autotune', type=lambda x:bool(strtobool(x)), default=False, nargs='?', const=True,
```

Comment:
Even if it doesn’t make much difference, consider turning on the auto tune by default